src/Client/IO/TezosClient.hs

```diff
@@ -160,6 +162,16 @@ getAddressForContract v tcPath alias  = do
         Left err -> throwM $ TzbtcParseError (show err))
       (\stderr' -> pure $ TzbtcTezosClientError stderr')
 
+getChainId
+  :: Bool -> FilePath -> Text -> IO (Either TzbtcClientError ChainId)
+getChainId v tcPath name  = do
+  callTezosClient' v tcPath
+    ["rpc", "get", "chains/" <> (toString name) <> "/chain_id"] Nothing (\stdout' ->
```

Comment:
Why are you doing it via `tezos-client` instead of querying node directly?

Reply:
It appeared to be a bit easier. But yes, this is an RPC call, so better done via our RPC infra.

Reply:
@gromakovsky Have moved it to RPC.